crates/ruff_linter/src/rules/pylint/rules/unnecessary_list_index_lookup.rs

```diff
@@ -4,15 +4,18 @@ use ruff_diagnostics::{AlwaysFixableViolation, Diagnostic, Edit, Fix};
 use ruff_macros::{derive_message_formats, violation};
 use ruff_python_ast::visitor;
 use ruff_python_ast::visitor::Visitor;
+use ruff_python_semantic::SemanticModel;
 use ruff_text_size::TextRange;
 
 use crate::checkers::ast::Checker;
 
 /// ## What it does
-/// Checks for access of a list item at the current index when using enumeration.
+/// Checks for index-based list accesses during `enumerate` iterations.
 ///
 /// ## Why is this bad?
-/// It is more succinct to use the variable for the value at the current index which is already in scope from the iterator.
+/// When iterating over a list with `enumerate`, the current item is already
+/// available alongside its index. Using the index to look up the item is
+/// unnecessary.
```

Comment:
I try to wrap these at 80 characters so that they render reasonably in the terminal.